AirCasting/SessionViews/SingleMeasurementView.swift

```diff
@@ -7,7 +7,7 @@ import Resolver
 
 struct SingleMeasurementView: View {
     @ObservedObject var stream: MeasurementStreamEntity
-    var threshold: SensorThreshold?
+    @ObservedObject var threshold: SensorThreshold
```

Comment:
2/2 You have deleted the optional and now it cannot be nil. When you look below in this code, there is a possibility that it can be (`if let threshold = threshold, measurementPresentationStyle == .showValues `{), we should stay with that approach as I believe it was done intentionally by someone